Article/Article.md

```diff
@@ -17,89 +17,41 @@ abstract: |
+The study of gender differences on a world scale per se is challenging. One of the challenges that hamper the progress is the lack of large and heterogeneous datasets across different social groups and countries. The Gallup World Poll 2012 included a Global Preference Survey conducted on almost 80000 people in 76 countries all around the world that aimed to fill this gap: Covering nearly 90% of the world population representation, with each country having around 1000 participants answering questions related to their time preference (patience), altruism, will of risk-taking, negative and positive reciprocity, and trust. The dataset provides a unique insight into the economic preferences of a heterogeneous number of people. The original study published in the Quarterly Journal of Economics [@QJE_Falk, 133 (4) pp. 1645-1692] focused on more general questions about the economic preferences distributions in different countries, trying to explore several covariates from the Gallup World Poll. While, the subsequent article [REF], replicated in this work, focused explicitly on the gender differences arising from the previous study and reported the evidence for the relationship between gender differences in economic preferences, economic development, and gender equality across many countries. The data reveals a positive correlation of gender differences in preferences with GDP p/c and with the gender equality of the countries, thus favoring the hypotheses that predict the increase of the differences as women and men obtain sufficient access to the resources to develop and express their intrinsic preferences independently.
 
-NOTE: Here I need to add something about our extended analysis
+Throughout the study [REF], the authors construct and use a joint measure of gender equality in different countries, 
+combining the indexes provided by WEF and UN, the female to male labor participation ratio, and the years since woman suffrage. However, no investigation and discussion on the validity of such an approach were provided. Moreover, a large body of literature on the subject was ignored. The last fact is critical as WEF and UN indicators are far from a complete representation of gender inequality and were heavily criticized by many authors. In this article, we argue that it is nearly impossible to conduct inferences about gender (in)equality and its relation with other socio-economic parameters, such as preferences, without a detailed understanding of the specifics of indicators that are used as proxies for gender equality in a domain of interest. Finally, we replicate the original article and show that [results here].
```

Comment:
A bit too much critical, in my opinion, I would moderate. Plus, citations needed